vmm/src/lib.rs

```diff
@@ -891,6 +900,53 @@ impl ReceiveAdditionalConnections {
         Ok((event_fd.try_clone()?, event_fd))
     }
 
+    /// Handle read requests. This function returns until the abort_event_fd is
+    /// triggered or the connection is closed or encountered an error.
+    fn abortable_read_requests(
```

Comment:
from the name of the function, I don't really understand what it is doing

Reply:
It's now `handle_requests`.